packages/block-editor/src/components/block-breadcrumb/index.js

```diff
@@ -60,7 +65,13 @@ function BlockBreadcrumb( { rootLabelText } ) {
 					<Button
 						className="block-editor-block-breadcrumb__button"
 						variant="tertiary"
-						onClick={ clearSelectedBlock }
+						onClick={ () => {
+							clearSelectedBlock();
+							// Focus the editor iframe.
+							blockRef.current
+								?.closest( '[aria-label="Block canvas"]' )
```

Comment:
Any reason why we don't just create a ref for `FocusableWrapper` and pass it down via context?

Reply:
That's a good idea -- I think we're going to try to focus the editor region instead though.

Reply:
I added a ref to do this. The downside is that the ref needs to be passed through props.